x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -360,17 +396,17 @@ const ProcessEventDotComponents = React.memo(
         return [];
       }
       // If we have entries to show, map them into options to display in the selectable list
-      return Object.entries(relatedStats).map((statsEntry) => {
-        const displayName = getDisplayName(statsEntry[0]);
+      return Object.entries(relatedStats).map(([name, value]) => {
```

Comment:
I see that this is older code. Any idea why we're using `Object.entries` here? We control the data structure (for the most part) so maybe we could change this.

Reply:
Nope, but I can update it to just a single iteration. 👍